source/codegen/metadata/nidaqmx/enums.py

```diff
@@ -1797,6 +1760,26 @@
             {
                 'name': 'CHANNEL_CURRENT',
                 'value': 1
+            },
+            {
+                'name': 'CHANNEL_HIGH_IMPEDANCE',
+                'value': 2
+            }
+        ]
+    },
+    'PowerUpStates': {
```

Comment:
These are the names from the internal daq api metadata. They aren't used in the C header, but do have the same names listed as "civName" and "lvName". So maybe it's public somewhere.

Reply:
I see them here: https://zone.ni.com/reference/en-XX/help/370469AP-01/lvdaqmx/mxsetpwrupstates/#DAQmxSetPowerUpStates(Digital)

So it's public in the documentation at least.